fapi-client/src/main/scala/com/gu/facia/api/utils/ResolvedMetaData.scala

```diff
@@ -59,8 +59,7 @@ object ResolvedMetaData {
       cardStyle match {
         case com.gu.facia.api.utils.Comment => Default.copy(
           showByline = true,
```

Comment:
`showByline` being `true` makes me really nervous as the `byline` field is optional in [the model](https://github.com/guardian/flexible-model/blob/2c9268bbceb4212de480ee47158f4b08eb4e2682/src/main/thrift/content.thrift#L1165-L1166)

Given this scenario is rare, would taking a more holistic view (i.e. remove all default metadata logic from the shared library) be better? That is, move to a place where the tooling is the canonical source of all metadata.

Reply:
yeah. I agree, if we are removing styling choices from here, we should probably remove all styling choices. It is a weird place for them.